src/ocamlorg_package/lib/ocamlorg_package.ml

```diff
@@ -386,6 +386,12 @@ let latest_documented_version t name =
   | None -> Lwt.return None
   | Some vlist -> aux (List.rev vlist)
 
+let is_latest_version t name version =
+  match get_package_versions t name with
```

Comment:
Unless I’ve missed another List.rev somewhere else I think this is missing. `Map.bindings` returns the list in increasing version number but you need the decreasing version number here
```suggestion
  match List.rev (get_package_versions t name) with
```

Reply:
The expected behaviour is to get the latest version of the package which is assumed to be the first of the list. Since we are trying to use pre-released versions as latest but also want to make with the most recent stable release, will reversing the entire list fix that? as the entire list of version would then have the oldest releases at the top and the most recent one at the bottom

Reply:
I’m not sure to get that. Have you tested this function? I haven’t but just reading `get_package_versions` it seems to be the inverse of what you want (the first element of the list is the oldest version not the latest)